src/OrchardCore/OrchardCore.Users.Core/Services/UserStore.cs

```diff
@@ -524,5 +526,9 @@ public async Task<IList<IUser>> GetUsersForClaimAsync(Claim claim, CancellationT
         }
 
         #endregion IUserClaimStore<IUser>
+
+        #region IQueryableuserStore
+        public IQueryable<IUser> Users => _session.Query<User>().ListAsync().GetAwaiter().GetResult().AsQueryable();
+        #endregion
```

Comment:
also here should be `.Query<User,UserIndex>()`

Reply:
This mean I should use what I suggested earlier [here](https://github.com/OrchardCMS/OrchardCore/issues/5916#issuecomment-612624587)

Reply:
@deanmarcussen @hishamco 

Sorry, this is me that i wrongly suggested to use `_session.Query<User>()` if there is no predicate ;)

But yes, using the indexed table is still much more better as it holds all the related ids, maybe i was too tired ;)